src/table/settings/gui_settings.ini

```diff
@@ -130,12 +130,17 @@ def      = false
 str      = STR_CONFIG_SETTING_SMOOTH_SCROLLING
 strhelp  = STR_CONFIG_SETTING_SMOOTH_SCROLLING_HELPTEXT
 
-[SDTC_BOOL]
-var      = gui.right_mouse_wnd_close
-flags    = SF_NOT_IN_SAVE | SF_NO_NETWORK_SYNC
-def      = false
+; This was originally right_mouse_wnd_close, renamed to change type to uint8
```

Comment:
This comment doesn't actually add value, funny enough. It is useful for this PR, sure, but once it is merged, it is no longer useful. Imagine if we leave everything we once removed in comments behind, the codebase would be littered with those comments :)

For this we use version-control. If someone actually wants to know these things, they can dive into the old history themselves :)